src/WebApp/flask/aztk_cluster.py

```diff
@@ -74,12 +74,16 @@ def createCluster(self):
             jars=jars
         )
 
+        clusterDetails = self.table_service.get_entity('cluster', 'predictivemaintenance', 'predictivemaintenance')
+        cluster_number = int(clusterDetails.ClusterNumber) + 1
```

Comment:
why not generate a unique string instead of incrementing?

Reply:
Did not want to exceed the length(as I was not sure what the requirements are for cluster name). I will test that adding a unique string